operators/pkg/instctrl/cloudinit.go

```diff
@@ -86,6 +94,41 @@ func (r *InstanceReconciler) EnforceCloudInitSecret(ctx context.Context) error {
 	return nil
 }
 
+// Get the specs of the volume
+func (r *InstanceReconciler) GetNfsSpecs(ctx context.Context) (serverIp, serverName, path string, err error) {
+	tenant := clctx.TenantFrom(ctx)
+	secretName := types.NamespacedName{Namespace: tenant.Namespace, Name: tntctrl.PvcSecretName}
```

Comment:
I'm not sure this is really needed: given the basic CrownLabs structure, instances should be already created within the tenant namespace, this retrieving the NS from the Instance itself would be enough. @giorio94 is this correct?

Reply:
Now using the instance namespace as it was not correct, the tenant namespace is ""

Reply:
Yes, also because the Tenant is a cluster wide resource, so the Namespace field (that likely comes from Metadata.Namespace) is empty, not being namespaced. The field you were looking for is more likely inside the tenant Status, but again, I'd not look for that and stick to retrieving the secret from the same NS of the instance.